debugging/1.0/_index.md

```diff
@@ -0,0 +1,52 @@
+<!-- Template for the root page of a specification release -->
+---
+title: "Debugging Support for Other Languages Spec 1.0"
+date: 2019-07-18T16:10:38-04:00
+summary: "First Version Spec for DSOL"
+---
+Jakarta Debugging Support for Other Languages provides a mechanism by which 
+programs executed under the Java virtual machine but written in languages 
+other than the Java programming language, can be debugged with references to
+the original source (for example, source file and line number references).
+
+* [Jakarta Debugging Support for Other Languages 1.0 Specification Document](./jdsol_1.0.pdf) (PDF)
+* [Jakarta Debugging Support for Other Language 1.0 Specification Document](./jdsol_1.0.html) (HTML)
+* [Jakarta  Debugging Support for Other Language 1.0 Javadoc](./apidocs)
+
```

Comment:
Need to add an entry for the standalone TCK:
* Jakarta Debugging Support for Other Languages TCK
with a link to:
https://download.eclipse.org/jakartaee/debugging/1.0/eclipse-debugging-tck-1.0.0.zip

Reply:
Hi Bill, the link is not found, can you help to check?

Reply:
The link is not found because that's the URL where the TCK will be published
**after it is approved**.